firmware/hw_layer/drivers/can/can_msg_tx.h

```diff
@@ -112,9 +112,9 @@ class CanTxTyped final : public CanTxMessage
 };
 
 template <typename TData>
-void transmitStruct(uint32_t id, bool isExtended)
+void transmitStruct(uint32_t id, bool isExtended, bool canChannel = false)
```

Comment:
shouldn't this be a `size_t` ("`bus`")?

line 31:
```c++
explicit CanTxMessage(uint32_t eid, uint8_t dlc = 8, size_t bus = 0, bool isExtended = false);
```

Reply:
let's make this parameter `canChannel` non-optional - that's sort of what got us in this mess in the first place

Reply:
> so that inside the function we look at which bus to use?

eh? All I mean by my comment was to remove `= false` on line 115 here.

Reply:
When I change that, I will update all uses for `transmitStruct`